lib/astarte_core/triggers/policy/range_error.ex

```diff
@@ -0,0 +1,49 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2021 Ispirata Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.Core.Triggers.Policy.RangeError do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Astarte.Core.Triggers.Policy.RangeError
+
+  @error_range 400..599
+
+  embedded_schema do
+    field :error_codes, {:array, :integer}
+  end
+
+  def validate(changeset) do
+    changeset
+    |> validate_subset(:error_codes, @error_range)
```

Comment:
Since the default error message is `has an invalid entry` I would help the user a little bit more by adding the option `message: "contains an invalid HTTP error code"` or something similar